packages/core/src/lib/TwaManifest.ts

```diff
@@ -154,6 +156,7 @@ export class TwaManifest {
   shareTarget?: ShareTarget;
   orientation: Orientation;
   fingerprints: Fingerprint[];
+  serviceAccountJson: string | undefined;
```

Comment:
Do we expect a JSON string or is this a reference to a JSON file?

Reply:
Looking at tests, it seems it's a reference to the JSON file. So, maybe we could call the field `serviceAccountJsonFile`?

Reply:
Done